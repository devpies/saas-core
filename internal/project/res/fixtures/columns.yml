# Project 1
- column_id: a4a420db-c6d6-4209-af96-5e5408a57bfe
  project_id: 96c3424e-17cf-4bd2-916e-1ec2ddc979a5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: To Do
  column_name: column-1
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.211989
  created_at: 2022-07-09 05:51:24.211989

- column_id: 4ffd92ae-32c7-4314-b295-18aa7da66d62
  project_id: 96c3424e-17cf-4bd2-916e-1ec2ddc979a5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: In Progress
  column_name: column-2
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.219037
  created_at: 2022-07-09 05:51:24.219037

- column_id: 29c1b8d2-caba-4d61-9523-a4d2b83df3d9
  project_id: 96c3424e-17cf-4bd2-916e-1ec2ddc979a5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: Review
  column_name: column-3
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.220731
  created_at: 2022-07-09 05:51:24.220731

- column_id: 37bd403f-a9dd-4527-a168-b3b2ac5666db
  project_id: 96c3424e-17cf-4bd2-916e-1ec2ddc979a5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: Done
  column_name: column-4
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.222499
  created_at: 2022-07-09 05:51:24.222499

# Project 2
- column_id: 67459719-a22d-4aff-a6ad-7216bbd87dbd
  project_id: f8a6daf8-7239-47c3-a4e7-74d46439c7e5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: To Do
  column_name: column-1
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.211989
  created_at: 2022-07-09 05:51:24.211989

- column_id: aae5b0b4-3c31-4019-a1b5-783c01de6884
  project_id: f8a6daf8-7239-47c3-a4e7-74d46439c7e5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: In Progress
  column_name: column-2
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.219037
  created_at: 2022-07-09 05:51:24.219037

- column_id: 131f311f-6880-44df-818b-0e986fbed104
  project_id: f8a6daf8-7239-47c3-a4e7-74d46439c7e5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: Review
  column_name: column-3
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.220731
  created_at: 2022-07-09 05:51:24.220731

- column_id: 83d3a5fd-08e2-487a-a882-a19717e8b143
  project_id: f8a6daf8-7239-47c3-a4e7-74d46439c7e5
  tenant_id: f24d653a-f465-11ec-bfa4-26b2e5d16858
  title: Done
  column_name: column-4
  task_ids: RAW='{}'
  updated_at: 2022-07-09 05:51:24.222499
  created_at: 2022-07-09 05:51:24.222499