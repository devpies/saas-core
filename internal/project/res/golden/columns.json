[
  {
    "id": "a4a420db-c6d6-4209-af96-5e5408a57bfe",
    "tenantID": "f24d653a-f465-11ec-bfa4-26b2e5d16858",
    "title": "To Do",
    "columnName": "column-1",
    "taskIds": [],
    "projectId": "96c3424e-17cf-4bd2-916e-1ec2ddc979a5",
    "updatedAt": "2022-07-09T05:51:24.211989Z",
    "createdAt": "2022-07-09T05:51:24.211989Z"
  },
  {
    "id": "4ffd92ae-32c7-4314-b295-18aa7da66d62",
    "tenantID": "f24d653a-f465-11ec-bfa4-26b2e5d16858",
    "title": "In Progress",
    "columnName": "column-2",
    "taskIds": [],
    "projectId": "96c3424e-17cf-4bd2-916e-1ec2ddc979a5",
    "updatedAt": "2022-07-09T05:51:24.219037Z",
    "createdAt": "2022-07-09T05:51:24.219037Z"
  },
  {
    "id": "29c1b8d2-caba-4d61-9523-a4d2b83df3d9",
    "tenantID": "f24d653a-f465-11ec-bfa4-26b2e5d16858",
    "title": "Review",
    "columnName": "column-3",
    "taskIds": [],
    "projectId": "96c3424e-17cf-4bd2-916e-1ec2ddc979a5",
    "updatedAt": "2022-07-09T05:51:24.220731Z",
    "createdAt": "2022-07-09T05:51:24.220731Z"
  },
  {
    "id": "37bd403f-a9dd-4527-a168-b3b2ac5666db",
    "tenantID": "f24d653a-f465-11ec-bfa4-26b2e5d16858",
    "title": "Done",
    "columnName": "column-4",
    "taskIds": [],
    "projectId": "96c3424e-17cf-4bd2-916e-1ec2ddc979a5",
    "updatedAt": "2022-07-09T05:51:24.222499Z",
    "createdAt": "2022-07-09T05:51:24.222499Z"
  }
]
